libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -98,6 +98,13 @@ const AP_Param::GroupInfo AP_MotorsMulticopter::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("THR_MIX_MAX", 14, AP_MotorsMulticopter, _thr_mix_max, AP_MOTORS_THR_MIX_MAX_DEFAULT),
 
+    // @Param: PWM_TYPE
+    // @DisplayName: Output PWM type
+    // @Description: This selects the output PWM type, allowing for normal PWM continuous output or OneShot125
+    // @Values: 0:Normal,1:OneShot,2:OneShot125
+    // @User: Advanced
+    AP_GROUPINFO("PWM_TYPE", 15, AP_MotorsMulticopter, _pwm_type, PWM_TYPE_NORMAL),
```

Comment:
they wait on pixhawk as it is a separate microcontroller doing the output, and a separate thread pushing to that micro.